### PR TITLE
dtx: adjust lof lot entry spacing per guide

### PR DESCRIPTION
**Pull request by alick**
work in progress. not ready for merge

### CODE REVIEW

**Comment by alick** [comment]
Should be OK for merge.

File: thuthesis.dtx [hunk 1]
```diff
@@ -2871,7 +2871,7 @@
     {\thu@chapter*[]{\csname list#1name\endcsname}\thu@starttoc{#1}}
     {\thu@chapter*{\csname list#1name\endcsname}\thu@starttoc{#1}}}
 \renewcommand\listoffigures{\thu@listof{figure}}
-\renewcommand*\l@figure{\@dottedtocline{1}{0em}{4em}}
+\renewcommand*\l@figure{\addvspace{6bp}\@dottedtocline{1}{0em}{4em}}
 \renewcommand\listoftables{\thu@listof{table}}
```
Comment (xueruini) [inline thuthesis.dtx]: #144 说表格和图的目录都有问题，这里怎么只改一个？

Comment (aixlx) [inline thuthesis.dtx]: >\renewcommand\listoffigures{\thu@listof{figure}}
\renewcommand*\l@figure{\addvspace{6bp}\@dottedtocline{1}{0em}{4em}}
\renewcommand\listoftables{\thu@listof{table}}
\let\l@table\l@figure

Comment (xueruini) [inline thuthesis.dtx]: > \let\l@table\l@figure

oh, got it.

**Pull request merged**